### PR TITLE
Fix Download

### PR DESCRIPTION
**Pull request by ericvn**

### CODE REVIEW

**Comment by geeknoid** [comment]
I thought the script has logic to figure out which version to download based on the branch name.

File: content/en/docs/setup/getting-started/index.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ Download the Istio release which includes installation files, samples, and the
     extract the latest release automatically:
 
     {{< text bash >}}
-    $ curl -L https://istio.io/downloadIstio sh -
+    $ curl -L https://istio.io/downloadIstio | ISTIO_VERSION=1.4.0 sh -
```
Comment (geeknoid) [inline content/en/docs/setup/getting-started/index.md]:
```suggestion
    $ curl -L https://istio.io/downloadIstio | ISTIO_VERSION={{< istio_full_version >}} sh -
```

**Comment by ericvn** [comment]
Nope, I had opened an issue and a PR to fix this, but nobody wanted to approve that at the time unfortunately: https://github.com/istio/istio/issues/17178

**Comment by geeknoid** [comment]
/cherrypick/master

**Comment by ericvn** [comment]
I assume https://istio.io/downloadIstio points to the release branch which doesn't have the fix to remove betas.

**Comment by geeknoid** [comment]
Correct, the redirect will always point to the current release branch.

**Pull request merged**